### PR TITLE
More verbose HTML mail layout

### PR DESCRIPTION
**Pull request by maximal**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

In order to make it better we should print app name, view title and character encoding but I don’t know how to make it in Yii3.

Yii2 equivalents:
* `<?= Html::encode(Yii::$app->name) ?>`
* `<?= Html::encode($this->title) ?>`
* `<?= Yii::$app->charset ?>`

### CODE REVIEW

File: resources/mail/layouts/html.php [hunk 1]
```diff
@@ -1 +1,20 @@
+<?php
+/* @var $content string Mail contents as view render result */
+?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>MAIL_VIEW_TITLE</title>
```
Comment (samdark) [inline resources/mail/layouts/html.php]: ?

File: resources/mail/layouts/html.php [hunk 2]
```diff
@@ -1 +1,20 @@
...
+</head>
+<body>
+
 <?= $content ?>
+
+<footer style="margin-top: 5em">
+-- <br>
+APP_NAME
```
Comment (samdark) [inline resources/mail/layouts/html.php]: ?

Comment (maximal) [reply resources/mail/layouts/html.php]: In order to make it better we should print app name, view title and character encoding but I don’t know how to make it in Yii3.

Yii2 equivalents:
* `<?= Html::encode(Yii::$app->name) ?>`
* `<?= Html::encode($this->title) ?>`
* `<?= Yii::$app->charset ?>`

**Comment by maximal** [comment]
My Yii2 template as an example:

image

File: resources/mail/layouts/html.php (hunk 2 above)
Comment (samdark) [reply resources/mail/layouts/html.php]: Set encoding to UTF 8. There's no application title in Yii 3 (yet).

Comment (maximal) [reply resources/mail/layouts/html.php]: Encoding already set, the title can be leaved constant for now or removed at all, as it isn’t displayed anywhere in mail agents.

Comment (samdark) [reply resources/mail/layouts/html.php]: Let's remove it.

Comment (maximal) [reply resources/mail/layouts/html.php]: Done.

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**